deployer/deployer.py

```diff
@@ -527,8 +540,6 @@ def deploy(self, environment='pre', eureka=True, health_check=None, health_check
             self.set_scheduler(version)
 
     def deploy_elb(self, health_check, health_check_port):
-        self.elbs = [self.app]
-
```

Comment:
I'm doing another PR to change this part because we need to add more then one elb in some cases... but I have problems with the commont tools to test it...